changelogs/fragments/54028_tower_settings_value_param_required.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "tower_settings - 'name' and 'value' parameters are always required, module can be used in order to get a setting"
```

Comment:
> module can be used in order to get a setting

module can _not_ be used to get a setting?

Reply:
Indeed, thanks !